app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/mysong/subfragment/artist/ArtistFragment.java

```diff
@@ -12,6 +12,7 @@
 import com.fstyle.androidtrainning.R;
 import com.fstyle.androidtrainning.model.Artist;
 import com.fstyle.androidtrainning.screen.BaseFragment;
+import com.fstyle.androidtrainning.screen.songbelongalbum.SongBelongAlbumActivity;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.screen.songbelongalbum.SongBelongAlbumActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports